LanguageFeatures/Static-access-shorthand/equality_A02_t02.dart

```diff
@@ -50,18 +52,28 @@ main() {
 // [analyzer] unspecified
 // [cfe] unspecified
 
-  if (m == condition ? M.intOne : .intTwo()) {}
-//                                ^
+  if ([m] == [.intOne]) {}
+//            ^
 // [analyzer] unspecified
 // [cfe] unspecified
 
-  if (m != condition ? .intTwo() : M.values[0]) {}
-//                     ^
+  if ({m} != {.intTwo()}) {}
+//            ^
 // [analyzer] unspecified
 // [cfe] unspecified
 
-  if (m != condition ? .values[0] : M.intTwo()) {}
-//                     ^
+  if ({"key": m} != {"key": .values[0]}) {}
+//            ^
```

Comment:
I guess it doesn't matter, but for consistency, perhaps?:
```suggestion
//                          ^
```
Or is there a reason why the error would actually be reported at `m`?

Reply:
No. Thank you! Fixed.